ob1k-core/src/main/java/com/outbrain/ob1k/client/dispatch/StaticDoubleDispatchStrategy.java

```diff
@@ -0,0 +1,27 @@
+package com.outbrain.ob1k.client.dispatch;
+
+import com.outbrain.ob1k.concurrent.Try;
+
+/**
+ * A simple double dispatch strategy with static threshold
+ *
+ * @author lifey
+ */
+public class StaticDoubleDispatchStrategy extends AbstractDoubleDispatchStrategy {
+
+  private final long durationMs;
+
+  public StaticDoubleDispatchStrategy(final long durationMs) {
+    this.durationMs = durationMs;
+  }
+
+  @Override
+  public long provideInterval() {
+    return durationMs;
+  }
+
+  @Override
+  public void dispatchResult(final Try result, final long startTimestamp) {
```

Comment:
dispatchResult is a bad name as it can be interpreted in two ways
1. the result of The dispatch 
2. take the result and dispatch it 
dispatchResultListener